package.json

```diff
@@ -1,41 +1,42 @@
 {
   "name": "generator-suit",
-  "version": "0.5.0",
+  "version": "1.0.0",
```

Comment:
Maybe change this and the `CHANGELOG` as a single `1.0.0` commit to follow convention?